tests/rfb-frames/test.rules

```diff
@@ -0,0 +1,3 @@
+alert rfb any any -> any any (msg:"RFB Frame 1" ;flow:to_client; frame:pdu; content:"|01 02|"; startswith; sid:1;)
+alert rfb any any -> any any (msg:"RFB Frame 2"; flow:to_server; frame:pdu; content:"|7A 6F 36|"; offset:2; sid:2;)
+alert rfb any any -> any any (msg:"RFB Frame 3"; frame:pdu; content:"|30 30 38 0A|"; endswith; sid:3;)
```

Comment:
Could we also have some not matching rules? e.g. content has more data that follows that packet frame. Should something like that be a match?
Others like not giving an offset but content is something from the middle of the pdu, etc.

Reply:
Do you mean adding the not matching rules and saying this rule should have 0 matching packets in the yaml file?

Reply:
> e.g. content has more data that follows that packet frame. Should something like that be a match?
Do you mean a wrong field of content?

Reply:
Yes.